modules/_node-scripts/index.mjs

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import getGlobalImports from './configuration/getGlobalImports.mjs';
+import getGlobalSymbols from './configuration/getGlobalSymbols.mjs';
+import getProjectDescription from './configuration/getProjectDescription.mjs';
+import getProjectExports from './configuration/getProjectExports.mjs';
+import getProjectMain from './configuration/getProjectMain.mjs';
+import getProjectNpmScriptsConfig from './configuration/getProjectNpmScriptsConfig.mjs';
+import getProjectWebContextPath from './configuration/getProjectWebContextPath.mjs';
+import writeCSSLoaderJavaScriptModules from './cssLoad/writeCSSLoaderJavaScriptModules.mjs';
+import bundleCSSExports from './esbuild/bundleCSSExports.mjs';
+import bundleJavaScriptExports from './esbuild/bundleJavaScriptExports.mjs';
+import bundleJavaScriptMain from './esbuild/bundleJavaScriptMain.mjs';
+import runNpmScripts from './npmscripts/runNpmScripts.mjs';
+import writeAMD2ESMBridges from './npmscripts/writeAMD2ESMBridges.mjs';
+import writeManifestJson from './npmscripts/writeManifestJson.mjs';
+import writePackageJson from './npmscripts/writePackageJson.mjs';
```

Comment:
Why are these specific to the npmscripts directory? Is it because they aren't esbuild specific? Perhaps we can have a new directory that is specific to liferay's needs. I would expect us to delete the npmscripts directory when we no longer need it.

Reply:
done